doc/guides/west/manifest.rst

```diff
@@ -775,63 +772,62 @@ hosted at ``https://git.example.com/mainline/manifest``.
      projects:
        - name: app
          url: https://git.example.com/mainline/app
-       - name: library
-         url: https://git.example.com/mainline/library
-         revision: refs/heads/only-in-mainline
-       - name: library2
-         url: https://git.example.com/mainline/library-2
-       - name: unnecessary-project
-         url: https://git.example.com/mainline/unnecessary-project
+       - name: lib
+         path: libraries/lib
+         url: https://git.example.com/mainline/lib
+       - name: lib2
+         path: libraries/lib2
+         url: https://git.example.com/mainline/lib2
 
    # downstream west.yml:
    manifest:
      projects:
        - name: mainline
          url: https://git.example.com/mainline/manifest
          import:
-           whitelist:
-             - library2
+           name-whitelist:
              - app
+             - lib2
            rename:
              app: mainline-app
-       - name: library2
-         path: mainline-lib2
        - name: app
          url: https://git.example.com/downstream/app
-       - name: library
-         url: https://git.example.com/downstream/library
+       - name: lib3
+         path: libraries/lib3
+         url: https://git.example.com/downstream/lib3
 
 An equivalent manifest in a single file would be:
 
 .. code-block:: yaml
 
    manifest:
      projects:
-       - name: library2
-         path: mainline-lib2
-         url: https://git.example.com/mainline/library-2
        - name: mainline-app
          url: https://git.example.com/mainline/app
+       - name: lib2
+         path: libraries/lib2
+         url: https://git.example.com/mainline/lib2
        - name: mainline
          url: https://git.example.com/mainline/manifest
        - name: app
          url: https://git.example.com/downstream/app
-       - name: library
-         url: https://git.example.com/downstream/library
+       - name: lib3
+         path: libraries/lib3
+         url: https://git.example.com/downstream/lib3
 
-If a whitelist had not been used:
+If a whitelist had not been used, the ``lib2`` project from the mainline
```

Comment:
Something is wrong here.
Cause you actually do import `lib2` as it is in the whitelist.
```
           name-whitelist:
             - app
             - lib
```
did you mean `lib` instead of `lib2` ?

Reply:
Eagle eye. Thanks for catching that. I did mean `lib`.